tests/test_bundle.py

```diff
@@ -2079,90 +2065,6 @@ def test_create_html_manifest():
     )
     assert single_file_index_file_ans == json.loads(manifest.get_flattened_copy().json)
 
-    # check all runtime_environment vars
-    single_file_index_file_ans = {
```

Comment:
What were these tests doing?

Reply:
These tests were all trying to confirm that R/Python environment and image fields were retained for static (HTML) content. We do not execute HTML content, so these fields are unnecessary. I removed them as inputs and removed the tests as appropriate.